### PR TITLE
add wavy route loader

### PR DESCRIPTION
**Pull request by dweill**
adds a new animated text loader for basically all the routes.

based on this: https://www.youtube.com/watch?v=m1ZKHPbnyjo with some tweaks for react to make it work.

### CODE REVIEW

File: villainary-react/app/profile/loading.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import WavyLoader from '../components/WavyLoader';
+
+export default function loading() {
+    return <WavyLoader text="mwahaha" />;
+}
```
Comment (christophermbooth) [inline villainary-react/app/profile/loading.tsx:5]: Why is this code duplicated so many times?  Is it possible to make this more dry?

File: villainary-react/app/globals.css [hunk 2]
```diff
@@ -3,21 +3,47 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.wavy {
+    position: relative;
+}
+
+.wavy span {
+    position: relative;
```
Comment (BethanyAJones) [inline villainary-react/app/globals.css:29]: Can these be alphabetized

File: villainary-react/app/profile/loading.tsx (hunk 1 above)
Comment (dweill) [reply villainary-react/app/profile/loading.tsx:5]: the way loading works in next js, is for each route, you have a loading.tsx file in the directory, and it automatically picks it up.

File: villainary-react/app/globals.css (hunk 2 above)
Comment (dweill) [reply villainary-react/app/globals.css:29]: sure

**Review by christophermbooth — APPROVED** [review]

**Pull request merged**